display_information/add-graphics-with-renderer/src/main/java/com/esri/samples/add_graphics_with_renderer/AddGraphicsWithRendererSample.java

```diff
@@ -54,13 +56,20 @@ public void start(Stage stage) {
       stage.setScene(scene);
       stage.show();
 
-      // create a map with topographic basemap
-      ArcGISMap map = new ArcGISMap(Basemap.Type.TOPOGRAPHIC, 15.169193, 16.333479, 2);
+      // authentication with an API key or named user is required to access basemaps and other location services
+      String yourAPIKey = System.getProperty("apiKey");
+      ArcGISRuntimeEnvironment.setApiKey(yourAPIKey);
 
-      // set the map to the view
+      // create a map with the topographic basemap style
+      ArcGISMap map = new ArcGISMap(BasemapStyle.ARCGIS_TOPOGRAPHIC);
+
+      // create a map view and set its map
```

Comment:
```suggestion
      // create a map view and set the map to it
```

Reply:
For reference, after discussing, we will update this comment to: `create a map view and set the map to it`